chapters/methodology/theory.tex

```diff
@@ -1,34 +1,53 @@
 \section{Theory: Angle-Informed Importance Maps for CADIS and FW-CADIS}
 \label{sec:methodtheory}
 
-There exists methods that have been developed to generate
+Methods exist that have been developed to generate
 variance reduction parameters
 for deep penetration radiation transport problems with strong anisotropy in the
 flux. These methods have shown to have varying success, and may not be fully
 automated. The solution proposed in this dissertation is a formulation that we
-have named the $\Omega$-methods. The foundational research
+have named the $\Omega$-methods. 
+% Maybe \Omega-CADIS-methods? \Omega-methods seems insufficiently specific...?
+The foundational research
 on which the $\Omega$-methods are built will be briefly described to begin this
 section. That discussion will serve as a primer for the subsequent section, which
 is an introduction to the $\Omega$-methods and a discussion on how they differ
 from their predecessors.
 
 \subsection{Previous Work}
+% I this whole section seems to be lit review. I'm not clear about why it is here.
+% I haven't read the lit review section, so maybe the framing makes more sense. 
+% This is why the big outline is helpful...
 \label{sec:omegabknd}
 
-As discussed in sections \ref{sec:CADIS}
+%A thing to note and think carefully about: we probably want to use the words
+%"speed up" about improving calculations. Both acceleration and optimization
+%have specific mathematical meaning; VR methods typically don't do either. I
+%bring this up bc it's something I did not used to be careful about and I got
+%dinged about it by reviewers. If we are careful with wording now then we won't
+%have to worry about anyone being nit-picky later (computational science; all
+%about precision...)
```

Comment:
"speed up" is a totally valid phrase and what is commonly used.

We're not choose a more optimal method. Optimization involves mathematical proof, which we're not doing. We're using a method we think will be better. I would use "better", "improved", anything like that. We're not proving anything is optimal, near optimal, etc.